tests/python/gaia-ui-tests/gaiatest/tests/brick/test_brick_verification.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def setUp(self):
+        # Open Firefox OS and Skip the First Time Use app
+        GaiaTestCase.setUp(self)
+        # Open the homescreen app
+        self.homescreen = Homescreen(self.marionette)
+
+    def test_wait_for_homescreen(self):
+        self.apps.switch_to_displayed_app()
+        self.homescreen.wait_for_homescreen_visible()
```

Comment:
I don't have a hard opinion about this, aside from that whatever we wait for needs to be something that indicated the phone booted up FxOS correctly. If one of the checks Dave suggests is better for that, works for me. If this is sufficient, it also works for me.

Being covered by a unit test isn't a concern here. The idea is this test stands alone and won't be changed by any other consideration other than changing the definition of what it means to be bricked. 

To be honest, if anything, I'd drop the dependency on the Homescreen app object entirely so that changes to that can't affect the test, though that would require maintaining its own selector. It's still pretty safe in that if the homescreen selector changes, everything breaks so that's pretty unlikely.